sdk/storage/azure-storage-file-datalake/CHANGELOG.md

```diff
@@ -3,7 +3,11 @@
 ## 12.6.1 (Unreleased)
 
 ### Features Added
-
+- Added support for `create_if_not_exists()` for the following clients:
+  - `BlobContainerClient`
```

Comment:
You should update each package's `CHANGELOG` individually with only the changes relevant to that that package. For example, this is the `CHAGENLOG` for datalake so it should just have ``Added support for `create_if_not_exists()` for `FileSystemClient`.`` Then you'd update each other package's `CHANGELOG` with the relevant client.